tests/ui/module_name_repetitions.rs

```diff
@@ -1,3 +1,5 @@
+// compile-flags: --test
```

Comment:
I couldn't find another attribute that caused this issue, that's why the `--test` is included here.